src/engraving/tests/keysig_data/keysig.mscx

```diff
@@ -69,7 +69,11 @@
             <sigD>4</sigD>
             </TimeSig>
           <Tempo>
-            <tempo>1.66667</tempo>
+            <playbackTempo>1.66667</playbackTempo>
+            <notatedTempo>100</notatedTempo>
+            <equation>q = 100</equation>
+            <equationVisible>1</equationVisible>
+            <followText>1</followText>
             <text>𝅘𝅥 = 100</text>
```

Comment:
Is there a reason for keeping the metronome marking in the `<text>` element now that you have the `<equation>` element?

Also, I originally suggested this for the MSCX syntax:

```xml
    <text>Some text <equation>q = 100</equation> some more text</text>
```

This allows text after the equation. Is there a reason you chose not to do it this way?